lib/pure/includes/osenv.nim

```diff
@@ -4,11 +4,14 @@ when not declared(os) and not declared(ospaths):
   {.error: "This is an include file for os.nim!".}
 
 from parseutils import skipIgnoreCase
+from sequtils import delete
```

Comment:
This import is not required, system.delete is good enough.

Reply:
I see.. I was actually surprised why it needed both start and end mandatorily :) I did not know of `system.delete`; will update.